packages/@aws-cdk/aws-s3-deployment/lib/bucket-deployment.ts

```diff
@@ -90,6 +90,17 @@ export interface BucketDeploymentProps {
    */
   readonly retainOnDelete?: boolean;
 
+  /**
+   * If this is set to false, when a Cloudfront invalidation request is made, the cloudformation deployment
+   * will not fail if errors are encountered during the waiter process.
+   *
+   * NOTICE: Configuring this to "false" might have operational implications. Please
```

Comment:
I presume that one of these operational implications is that users will have to wait until the files expire before they can see any change. Is this correct? In general, I think we should spell out all these implications here.

Reply:
Yes, that is the intent. That's a great call out and I can tweak the language.